coral/src/app/components/AuthenticationRequiredBoundary.tsx

```diff
@@ -21,7 +22,31 @@ class AuthenticationRequiredBoundary extends Component<{
   render() {
     if (this.state.hasError) {
       window.location.assign("/login");
-      return <div>Redirecting to login...</div>;
+      return (
+        <Flexbox
+          paddingTop={"l6"}
+          justifyContent={"center"}
+          alignItems={"center"}
+        >
+          <div
+            role="alertdialog"
+            aria-labelledby={"authentication-required-heading"}
+            aria-describedby={"authentication-required-text"}
+          >
+            <Typography.Heading color={"secondary-100"}>
+              <span id={"authentication-required-heading"}>
```

Comment:
What is the purpose of the `id` attribute? is it related to accessibiility?

Reply:
Yes! The div with `role="alertdialog"` has an `aria-labelledby` and an `aria-describedby` attribute. There I use the `id`s from the elements that contain the accessible label and description. That way, e.g. a screenreader know what to read out here related to the `alertdialog`.